stor/dx.py

```diff
@@ -1,32 +1,34 @@
 from cached_property import cached_property
 import six
+import warnings
 
 import dxpy
 import dxpy.exceptions as dx_exceptions
-from dxpy.exceptions import DXError
+from dxpy.exceptions import DXError, DXSearchError
 
 from stor import exceptions as stor_exceptions
 from stor import Path
 from stor import utils
 from stor.obs import OBSPath
+from stor.obs import OBSUploadObject
 
 
 DNAnexusError = stor_exceptions.RemoteError
 NotFoundError = stor_exceptions.NotFoundError
 
 
-def _parse_dx_error(exc, **kwargs):
-    """
-    Parses DXError exceptions to throw a more informative exception.
-    """
-    msg = exc.message
-    exc_type = type(exc)
-
-    if exc_type is dx_exceptions.DXSearchError:
-        if msg and 'found more' in msg.lower():
-            return DuplicateError(msg, exc)
-        elif msg and 'found none' in msg.lower():
-            return NotFoundError(msg, exc)
+# def _parse_dx_error(exc, **kwargs):
```

Comment:
I have not yet deleted this because I think I will need a parse_error function inside the wrapper I build to handle dxpy errors like authentication. Will delete in next iteration if I don't use it there.